src/components/Form/Textarea.tsx

```diff
@@ -0,0 +1,139 @@
+  textarea: `${baseClassName}_textarea`,
+  hintText: `${baseClassName}_hintText`,
+  statusText: `${baseClassName}_statusText`,
+};
+
+export interface TextareaProps extends HtmlTextareaProps {
+  /** Custom classname to extend or customize */
+  className?: string;
+  /** Disable usage */
+  disabled?: boolean;
+  /** Event handler to execute when clicked */
+  onClick?: () => void;
+  /** To execute on textarea text change */
+  onChange?: (event: ChangeEvent<HTMLTextAreaElement>) => void;
+  /** To execute on textarea text onBlur */
+  onBlur?: (event: FocusEvent<HTMLTextAreaElement>) => void;
+  /** Label */
+  labelText: string;
+  /** Hide or show label. Label element is always present, but can be visually hidden.
+   * @default visible
+   */
+  labelMode?: 'hidden' | 'visible';
+  /** Placeholder text for input. Use only as visual aid, not for instructions. */
+  visualPlaceholder?: string;
+  /** Text content for textarea */
+  children?: string;
+  /** Hint text to be shown below the component */
+  hintText?: string;
+  /**
+   * 'default' | 'error'
+   * @default default
+   */
+  status?: 'default' | 'error';
+  /** Status text to be shown below the component and hint text. Use e.g. for validation error */
+  statusText?: string;
+  /** Resize mode of the textarea
+      'both' | 'vertical' | 'horizontal' | 'none'
+      @default 'vertical' 
+   */
+  resize?: 'both' | 'vertical' | 'horizontal' | 'none';
+  optionalText?: string;
+}
+
+class BaseTextarea extends Component<TextareaProps> {
+  render() {
+    const {
+      id: propId,
+      className,
+      disabled = false,
+      children,
+      onClick,
+      labelMode,
+      labelText,
+      hintText,
+      status,
+      statusText,
+      visualPlaceholder,
+      resize: dismissResize,
+      optionalText,
+      ...passProps
+    } = this.props;
+
+    const onClickProps = !!disabled ? {} : { onMouseDown: onClick };
+
+    const hideLabel = labelMode === 'hidden';
+    const id = idGenerator(propId);
+    const statusTextId = `${idGenerator(propId)}-statusText`;
+    const hintTextId = `${idGenerator(propId)}-hintText`;
+
+    return (
+      <HtmlLabel className={classnames(baseClassName, className, {})}>
+        {hideLabel ? (
+          <VisuallyHidden>{labelText}</VisuallyHidden>
+        ) : (
+          <Paragraph>
+            <HtmlSpan className={textareaClassNames.label}>
+              {labelText}
+            </HtmlSpan>
+            {optionalText && (
+              <HtmlSpan className={textareaClassNames.optionalText}>
+                {`(${optionalText})`}
+              </HtmlSpan>
+            )}
+          </Paragraph>
+        )}
+        {hintText && (
+          <HtmlSpan className={textareaClassNames.hintText} id={hintTextId}>
+            {hintText}
+          </HtmlSpan>
+        )}
+        <HtmlTextarea
+          id={id}
+          className={textareaClassNames.textarea}
+          disabled={disabled}
```

Comment:
How does this work with screen readers and keyboards? Especially tab-navigation?

Reply:
It seems to work as expecting; when using Tab to move the focus to textarea it will read first the label (and status text and hint texts if provided) then the contents of the textarea itself.
To get out of the textarea, user can press Esc to exit "edit mode" to continue using Tab to navigate.

Reply:
Even if disabled={true}?

Reply:
No, just like **TextInput, Checkbox, Toggle and LanguageMenu (+ Radiobutton)** it can not have focus when it is disabled with Tab.
As this is also related to that if **Button** can receive focus when disabled, we should then adjust all to work the same way if so decided. 

Maybe do the change for all the components in the same PR to have them work the same when decisions have been made?

Reply:
Sure, just wanted to make sure. Maybe using the standard disabled is a better way to go since we cannot know all implications of using an alternative way.

Reply:
I propose we leave this decision to be made later on and stick with the current implementation for now.